test/api/bundle.test.ts

```diff
@@ -43,44 +45,33 @@ test('write', () => {
     packageDir: pkg.dir,
     entryPoints: [pkg.entrypoint],
     allowedLicenses: ['Apache-2.0', 'MIT'],
+    // makes the bundle snapshot determenistic
+    sourcemap: false,
   });
 
   const bundleDir = bundle.write();
 
   expect(fs.existsSync(path.join(bundleDir, pkg.entrypoint))).toBeTruthy();
   expect(fs.existsSync(path.join(bundleDir, 'package.json'))).toBeTruthy();
-  expect(fs.existsSync(path.join(bundleDir, 'THIRD_PARTY_LICENSES'))).toBeTruthy();
   expect(fs.existsSync(path.join(bundleDir, 'lib', 'foo.js'))).toBeTruthy();
   expect(fs.existsSync(path.join(bundleDir, 'lib', 'bar.js'))).toBeTruthy();
   expect(fs.existsSync(path.join(bundleDir, 'node_modules'))).toBeFalsy();
   expect(fs.existsSync(path.join(bundleDir, '.git'))).toBeFalsy();
 
   const manifest = fs.readJSONSync(path.join(bundleDir, 'package.json'));
+  const entrypoint = fs.readFileSync(path.join(bundleDir, pkg.entrypoint), { encoding: 'utf-8' });
 
+  expect(entrypoint).toMatchSnapshot();
+  expect(Object.keys(manifest.devDependencies)).toEqual(['dep1', 'dep2']);
   expect(manifest.dependencies).toEqual({});
 
 });
 
 test('pack', () => {
 
   const pkg = Package.create({ name: 'consumer', licenses: ['Apache-2.0'] });
-  const dep1 = pkg.addDependency({ name: 'dep1', licenses: ['MIT'] });
-  const dep2 = pkg.addDependency({ name: 'dep2', licenses: ['Apache-2.0'] });
-
-  const attributions = [
-    'The consumer package includes the following third-party software/licensing:',
-    '',
-    `** ${dep1.name}@${dep1.version} - https://www.npmjs.com/package/${dep1.name}/v/${dep1.version} | MIT`,
-    '',
-    '----------------',
-    '',
-    `** ${dep2.name}@${dep2.version} - https://www.npmjs.com/package/${dep2.name}/v/${dep2.version} | Apache-2.0`,
-    '',
-    '----------------',
-    '',
-  ];
```

Comment:
All of this is simply not needed, I was trying to manually create the expected attribution file so the test passes validation, instead of simply generating it as part of the test.